.cspell.json

```diff
@@ -164,6 +176,12 @@
         "source: .*"
       ]
     },
+    {
+      "filename": "**/{*.cmake,CMakeLists.txt}",
+      "ignoreRegExpList": [
+        "externalproject"
```

Comment:
FYI, you can add this to https://github.com/tier4/cspell-dicts/blob/main/cmake/words.txt instead.

Reply:
Thank you for the information!
I'll delete it and add it to cspell-docts.